components/TARGET_PSA/services/crypto/COMPONENT_SPE/psa_crypto_access_control.c

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2019, Arm Limited and affiliates
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <string.h>
+
+#include "psa_crypto_access_control.h"
+#include "psa_crypto_slot_management.h"
+#include "spm_panic.h"
+
+typedef struct psa_crypto_access_control_s {
+    psa_key_handle_t key_handle;
+    int32_t partition_id;
+} psa_crypto_access_control_t;
+
+static psa_crypto_access_control_t crypto_access_control_arr[PSA_KEY_SLOT_COUNT];
+
+#define PSA_CRYPTO_ACCESS_CONTROL_RESET() (memset(crypto_access_control_arr, 0, sizeof(crypto_access_control_arr)))
```

Comment:
you have 
psa_crypto_access_control_reset() function no?

Reply:
The macro was removed in 6475bc65d818fc9ff4fc6c2c02de3d1b7ecde24a in favor of the static inline function.